package.xml

```diff
@@ -0,0 +1,22 @@
+<package format="2">
+  <name>dynamic-graph</name>
+  <version>3.7.3</version>
+  <description>
+    Dynamic graph library
+  </description>
+  <maintainer email="ostasse@laas.fr">Olivier Stasse</maintainer>
+  <license>BSDv2</license>
+
+  <url>http://github.com/stack-of-tasks/dynamic-graph</url>
+  <author>Nicolas Mansard</author>
+  <author>Olivier Stasse</author>
+
+  <build_depend>roscpp</build_depend>
+
+  <exec_depend>roscpp</exec_depend>
```

Comment:
I don't think `roscpp` is really required to build or exec this package, but it may be if we consider only catkin builds, I don't know.